libdfu/meson.build

```diff
@@ -89,41 +89,43 @@ executable(
   install_dir : get_option('bindir')
 )
 
-gnome.generate_gir(dfu,
-  sources : [
-    'dfu-common.c',
-    'dfu-common.h',
-    'dfu-context.c',
-    'dfu-context.h',
-    'dfu-device.c',
-    'dfu-device.h',
-    'dfu-element.c',
-    'dfu-element.h',
-    'dfu-error.c',
-    'dfu-error.h',
-    'dfu-firmware.c',
-    'dfu-firmware.h',
-    'dfu-image.c',
-    'dfu-image.h',
-    'dfu-sector.c',
-    'dfu-sector.h',
-    'dfu-target.c',
-    'dfu-target.h',
-  ],
-  nsversion : '1.0',
-  namespace : 'Dfu',
-  symbol_prefix : 'dfu',
-  identifier_prefix : 'Dfu',
-  export_packages : 'dfu',
-  dependencies : deps,
-  includes : [
-    'AppStreamGlib-1.0',
-    'Gio-2.0',
-    'GObject-2.0',
-    'GUsb-1.0',
-  ],
-  install : true
-)
+if get_option('enable-gir')
```

Comment:
It's probably obvious, but once you've got the appstream-glib one merged, make sure you align the parameter name between the two.